package.json

```diff
@@ -83,6 +83,7 @@
     "build": "vite build",
     "e2e:client": "cypress run",
     "e2e:gui": "cypress open",
+    "e2e:gui:playground": "cypress open --config-file tests/e2e/playground/playground.config.js",
```

Comment:
Command to open cypress UI on the playground tests. This way they stay out of the normal use-case, but can be more easily accessed when you want them